tsconfig.json

```diff
@@ -1,5 +1,10 @@
 {
   "compilerOptions": {
+    "baseUrl": "src",
+    "paths": {
+      "@pages/*": ["pages/*"],
```

Comment:
let's do it this way:
```suggestion
      "@/*": ["src/*"],
```
it is more flexible because you do not need to add aliases for each folder.

Reply:
also update `vite.config.ts`